packages/docsearch-react/src/DocSearch.tsx

```diff
@@ -16,17 +15,23 @@ export type DocSearchTranslations = Partial<{
   modal: ModalTranslations;
 }>;
 
+export type DocSearchTransformClient = {
```

Comment:
```suggestion
// The interface that describes the minimal implementation required for the algoliasearch client, when using the [`transformSearchClient`](https://docsearch.algolia.com/docs/api/#transformsearchclient) option
export type DocSearchTransformClient = {
```

wdyt?

we also need to update the definition here https://docsearch.algolia.com/docs/api/#transformsearchclient

Reply:
good point